src/ExampleData.js

```diff
@@ -2,191 +2,191 @@ export const exampleData = [
   {
     banner: "//cdn.minecraft-server-list.com/serverlogo/268353.jpg",
     name: "LemonCloud Minecraft Server",
-    ip: "play.lemoncloud.net"
+    domain: "play.lemoncloud.net"
```

Comment:
this should probably be address, all references to domain should also be address